patches/minecraft/net/minecraft/entity/item/EntityMinecart.java.patch

```diff
@@ -223,7 +236,15 @@
                          p_70108_1_.field_70159_w *= (double)0.2F;
                          p_70108_1_.field_70179_y *= (double)0.2F;
                          p_70108_1_.func_70024_g(this.field_70159_w + d0, 0.0D, this.field_70179_y + d1);
-@@ -782,6 +796,32 @@
+@@ -746,6 +759,7 @@
+       return this.field_70180_af.func_187225_a(field_184266_b);
+    }
+ 
++   @Deprecated // Forge: Use individual minecart methods instead
+    public abstract EntityMinecart.Type func_184264_v();
```

Comment:
Enhances mod compatibility as we discourage using `Type` enum for minecarts but use methods instead

Reply:
In the comment, please give examples for the methods that should be used instead.